portfolio/src/main/webapp/style.css

```diff
@@ -6,4 +6,18 @@
 
 #greeting-container {
```

Comment:
id should not be used as a selector. Only classes should be used.

Ex. ".GreetingContainer".

Note that the class name should be in title case only.

This is because the primary purpose of id is used for selecting elements in JS.